minifi/minifi-toolkit/minifi-toolkit-encrypt-config/src/main/java/org/apache/nifi/minifi/toolkit/config/command/MiNiFiEncryptConfig.java

```diff
@@ -40,54 +53,76 @@
 import picocli.CommandLine.Option;
 
 /**
- * Shared Encrypt Configuration for NiFi and NiFi Registry
+ * Encrypt Configuration for MiNiFi
  */
 @Command(
-        name = "encrypt-config",
-        sortOptions = false,
-        mixinStandardHelpOptions = true,
-        usageHelpWidth = 160,
-        separator = " ",
-        version = {
-                "Java ${java.version} (${java.vendor} ${java.vm.name} ${java.vm.version})"
-        },
-        descriptionHeading = "Description: ",
-        description = {
-                "encrypt-config supports protection of sensitive values in Apache MiNiFi"
-        }
+    name = "encrypt-config",
+    sortOptions = false,
+    mixinStandardHelpOptions = true,
+    usageHelpWidth = 160,
+    separator = " ",
+    version = {
+        "Java ${java.version} (${java.vendor} ${java.vm.name} ${java.vm.version})"
+    },
+    descriptionHeading = "Description: ",
+    description = {
+        "encrypt-config supports protection of sensitive values in Apache MiNiFi"
+    }
 )
-public class MiNiFiEncryptConfig implements Runnable{
+public class MiNiFiEncryptConfig implements Runnable {
 
     static final String BOOTSTRAP_ROOT_KEY_PROPERTY = "minifi.bootstrap.sensitive.key";
 
     private static final String WORKING_FILE_NAME_FORMAT = "%s.%d.working";
     private static final int KEY_LENGTH = 32;
 
     @Option(
-            names = {"-b", "--bootstrapConf"},
-            description = "Path to file containing Bootstrap Configuration [bootstrap.conf] for optional root key and property protection scheme settings"
+        names = {"-b", "--bootstrapConf"},
+        description = "Path to file containing Bootstrap Configuration [bootstrap.conf] for optional root key and property protection scheme settings"
     )
     Path bootstrapConfPath;
 
     @Option(
-            names = {"-B", "--outputBootstrapConf"},
-            description = "Path to output file for Bootstrap Configuration [bootstrap.conf] with root key configured"
+        names = {"-B", "--outputBootstrapConf"},
+        description = "Path to output file for Bootstrap Configuration [bootstrap.conf] with root key configured"
     )
     Path outputBootstrapConf;
 
+    @Option(
```

Comment:
Could you update the minifi/minifi-toolkit/minifi-toolkit-assembly/README.md with the new flags? An maybe worth to mention that the nifi.minifi.sensitive.props.key, nifi.minifi.sensitive.props.algorithm properties needs to be set

Reply:
Sure, added the instructions to the guide